lib/pengine/group.c

```diff
@@ -136,11 +136,15 @@ group_header(pcmk__output_t *out, int *rc, const pe_resource_t *rsc,
     }
 
     if (attrs != NULL) {
-        PCMK__OUTPUT_LIST_HEADER(out, FALSE, *rc, "Resource Group: %s (%s)",
-                                 rsc->id, (const char *) attrs->str);
+        PCMK__OUTPUT_LIST_HEADER(out, FALSE, *rc, "Resource Group: %s (%s) %s",
+                                 rsc->id,
+                                 desc ? desc : "",
+                                 (const char *) attrs->str);
         g_string_free(attrs, TRUE);
     } else {
-        PCMK__OUTPUT_LIST_HEADER(out, FALSE, *rc, "Resource Group: %s", rsc->id);
+        PCMK__OUTPUT_LIST_HEADER(out, FALSE, *rc, "Resource Group: %s %s",
+                                rsc->id,
+                                desc ? desc : "");
```

Comment:
I think you meant desc to be after attrs in the first case. attrs->str should still be in parentheses. desc should be in separate parentheses so we'll need the usual %s%s%s trick for it